bitnami/mariadb/templates/serviceaccount.yaml

```diff
@@ -9,4 +9,8 @@ metadata:
     chart: "{{ template "mariadb.chart" . }}"
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
```

Comment:
Please, could we use this helper instead? 

https://github.com/bitnami/charts/blob/81e98b36f6c4e8322a73d4220eb133f00f81ec1d/bitnami/mariadb/templates/_helpers.tpl#L280